airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java

```diff
@@ -208,6 +208,7 @@ public AirbyteCatalog discover(final JsonNode config) throws Exception {
     // create it.
     final AirbyteConnectionStatus check = check(config);
 
+    //If we throw an exception here,the close method of the tunnel will never be called cause iterator has not been invoked yet
```

Comment:
Does it make sense to put just a comment like this here and not do anything about the exception thrown below on line 213?

or is that exception  now being handled properly in `airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/ssh/SshWrappedSource.java`? thus, maybe the phrasing of the comment should be tweaked?